changelog/highcharts/11.1.0.md

```diff
@@ -1,23 +1,9 @@
-# Changelog for Highcharts v11.1.0 (2023-06-05)
+# Changelog for Highcharts v11.1.0 (2023-06-29)
```

Comment:
Other products than Dashboards should not change. Was the plan to include Highcharts Core also in the Dashboards changelog?